### ISSUE TITLE
School View next isnt working 

### ISSUE THREAD

**Issue by srujan21** [tpruvot/fullcalendar#24]
Hello , the school view isn't working properly. Its skipping a year every time I click Next.

**Comment by tpruvot** [comment]
indeed, i see that, maybe a side effect of december.. but i dont maintain this project, was just a sample mod

**Comment by srujan21** [comment]
@tpruvot Is there any way to fix it, I m using that in production and the fix is very important for me. Thanks :)
